openedx/core/djangoapps/user_authn/api/views.py

```diff
@@ -65,12 +65,12 @@ def get(self, request, **kwargs):  # lint-amnesty, pylint: disable=unused-argume
                     'fields': registration_fields,
                     'extended_profile': configuration_helpers.get_value('extended_profile_fields', []),
                 })
-            optional_fields = RegistrationFieldsContext('optional')._get_fields()  # pylint: disable=protected-access
-            if optional_fields:
-                context['optional_fields'].update({
-                    'fields': optional_fields,
-                    'extended_profile': configuration_helpers.get_value('extended_profile_fields', []),
-                })
+                optional_fields = RegistrationFieldsContext('optional')._get_fields()  # pylint: disable=protected-access
```

Comment:
Can we make `_get_fields()` a non protected function? i.e `get_fields()`

Reply:
@zainab-amir I have addressed your comments